src/main/java/org/folio/inventory/validation/experimental/InProcessNonRequestableTargetValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import java.util.Set;
+
+import static java.util.Set.of;
+
+public class InProcessNonRequestableTargetValidator extends AbstractTargetValidator {
+
+  public InProcessNonRequestableTargetValidator() {
+    super(ItemStatusName.IN_PROCESS_NON_REQUESTABLE, of(
+      ItemStatusName.AVAILABLE,
+      ItemStatusName.AWAITING_DELIVERY,
+      ItemStatusName.AWAITING_PICKUP,
+      ItemStatusName.IN_TRANSIT,
+      ItemStatusName.LOST_AND_PAID,
+      ItemStatusName.MISSING,
+      ItemStatusName.ORDER_CLOSED,
+      ItemStatusName.PAGED,
+      ItemStatusName.WITHDRAWN
+    ));
+  }
+
```

Comment:
Please remove this redundant blank line, I don't think it improves the readability of the code